pic32/cores/pic32/WInterrupts.c

```diff
@@ -263,12 +265,22 @@ void __attribute__((nomips16,at_vector(_EXTERNAL_0_VECTOR),interrupt(IPL4SRS)))
 void __attribute__((interrupt(),nomips16)) ExtInt0Handler(void)
 #endif
 {
-
+	VEC_ENT_CODE(EXT_INT0,INT0IF,PIN_INT0);
```

Comment:
FWIW, RFI/EMI/EMC is a known hassel when dealing with the sort of thing you described above, and yes, I know that most processors check int states on instruction fetches... If you get toggles that are too fast, then the processor is too slow for what you are attempting to do -- all the more important to just handle it and not worry why... when the handler queries the attached hardware, it will find out the why, and if there is no why, it should be designed to deal with that situation. I've seen this situation before, and the way to solve it is to not use a hammer for a screw, you get a better tool, a faster processor.